controls/limit.sp

```diff
@@ -47,7 +47,12 @@ benchmark "limit" {
     control.compute_snapshot_label_limit,
     control.dns_managed_zone_label_limit,
     control.sql_database_instance_label_limit,
-    control.storage_bucket_label_limit
+    control.storage_bucket_label_limit,
+    control.bigtable_instance_label_limit,
+    control.dataproc_cluster_label_limit,
+    control.dataproc_job_label_limit,
+    control.pubsub_subscription_label_limit,
+    control.pubsub_topic_label_limit
```

Comment:
Please sort the controls with the others in the benchmark.

The sorted list should look something as follows:

```    control.bigquery_dataset_label_limit,
    control.bigquery_job_label_limit,
    control.bigquery_table_label_limit,
    control.bigtable_instance_label_limit,
    control.compute_disk_label_limit,
    control.compute_forwarding_rule_label_limit,
    control.compute_image_label_limit,
    control.compute_instance_label_limit,
    control.compute_snapshot_label_limit,
    control.dataproc_cluster_label_limit,
    control.dataproc_job_label_limit,
    control.dns_managed_zone_label_limit,
    control.pubsub_subscription_label_limit,
    control.pubsub_topic_label_limit,
    control.sql_database_instance_label_limit,
    control.storage_bucket_label_limit